src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -588,139 +588,114 @@ def test_graph_user_scenario(self):
+            'mail_nick_name': 'deleteme11',
+            'child_group': self.create_random_name(prefix='testchildgroup', length=24),
+            'leaf_group': self.create_random_name(prefix='testleafgroup', length=24),
+            'user1': self.create_random_name(prefix='testgroupuser1', length=24),
+            'user2': self.create_random_name(prefix='testgroupuser2', length=24),
+            'pass': 'Test1234!!',
+            'domain': domain
         }
         self.recording_processors.append(AADGraphUserReplacer('@' + domain, '@example.com'))
         try:
-            # create user1
-            user1_result = self.cmd('ad user create --display-name {user1} --password {pass} --user-principal-name {user1}@{domain}').get_output_in_json()
-            self.kwargs['user1_id'] = user1_result['id']
-
-            # update user1
-            self.cmd('ad user update --display-name {user1}_new --account-enabled false --id {user1}@{domain} --mail-nickname {new_mail_nick_name}')
-            user1_update_result = self.cmd('ad user show --upn-or-object-id {user1}@{domain}', checks=[self.check("displayName", '{user1}_new')]).get_output_in_json()
-            self.cmd('ad user update --id {user1}@{domain} --password {pass}')
-            self.cmd('ad user update --id {user1}@{domain} --password {pass} --force-change-password-next-login true')
-            with self.assertRaises(CLIError):
-                self.cmd('ad user update --id {user1}@{domain} --force-change-password-next-login false')
-            self.kwargs['user1_id'] = user1_update_result['id']
-
-            # create user2
-            user2_result = self.cmd('ad user create --display-name {user2} --password {pass} --user-principal-name {user2}@{domain}').get_output_in_json()
-            self.kwargs['user2_id'] = user2_result['id']
             # create group
-            group_result = self.cmd('ad group create --display-name {group} --mail-nickname {group} --description {group}').get_output_in_json()
+            group_result = self.cmd(
+                'ad group create --display-name {group} --mail-nickname {mail_nick_name} --description {group}',
+                checks=[self.check('displayName', '{group}'),
+                        self.check('mailNickname', '{mail_nick_name}'),
+                        self.check('description', '{group}')]
+            ).get_output_in_json()
             self.kwargs['group_id'] = group_result['id']
-            # add user1 into group
-            self.cmd('ad group member add -g {group} --member-id {user1_id}',
-                     checks=self.is_empty())
-            # add user2 into group
-            self.cmd('ad group member add -g {group} --member-id {user2_id}',
-                     checks=self.is_empty())
-
-            # show user's group memberships
-            self.cmd('ad user get-member-groups --upn-or-object-id {user1_id}',
-                     checks=self.check('[0].displayName', self.kwargs['group']))
```

Comment:
`az ad user` related tests are totally removed from `GraphGroupScenarioTest`
@calvinhzy for awareness